upgrade.sh

```diff
@@ -67,6 +67,28 @@ undeploy -n gatekeeper-system gatekeeper-constraints
 undeploy -n gatekeeper-system gatekeeper-policy-library
 undeploy -n gatekeeper-system gatekeeper
 
+# Checks for gatekeeper-system namespace on the system.
+# Delete if it is still present on the system.
+
+gatekeeper_namespace=$(kubectl get namespaces | grep -i gatekeeper-system)
```

Comment:
`set -exo pipefail` is set in this script. So, of this command should fail (the namespace doesn't exist), it will fail the upgrade script overall. Must be fixed.

Reply:
Changed the code and tested. Logs are as below,
ncn-m001:/tmp/pradeep/feb3 # kubectl create namespace gatekeeper-system
namespace/gatekeeper-system created
ncn-m001:/tmp/pradeep/feb3 # ./delNS.sh
gatekeeper-system   Active   3s
namespace "gatekeeper-system" deleted
ncn-m001:/tmp/pradeep/feb3 # ./delNS.sh
ncn-m001:/tmp/pradeep/feb3 #
ncn-m001:/tmp/pradeep/feb3 # cat delNS.sh
#!/bin/bash
set -eu 
# Checks if gatekeeper related clusterrole, rolebindings and namespace are deleted.
# Delete if still they are present on the system.
function delNSGatekeeper() {
    kubectl get namespaces | grep -i gatekeeper-system || return 0
    kubectl delete --ignore-not-found=true namespace gatekeeper-system
} 
delNSGatekeeper
ncn-m001:/tmp/pradeep/feb3 #